event/events.go

```diff
@@ -118,5 +135,25 @@ func GenerateSubId() (string, error) {
 	}
 	rStr := hex.EncodeToString(b)
 	return strings.ToUpper(rStr), nil
+}
 
+func mapToOurEventData(eventData anyEventData) (txs.EventData, error) {
+	// While we depend on go-events in the way we do, we don't have much choice
```

Comment:
Well, I fully agree this is a problematic construction and we need to sort it out first.  But this is why I committed a proposal base code (copied from go-ethereum) in `/event_new`; because the only solution I see is building this event stream from the ground up.  In my opinion, you're right to address this like this, but it's a temporary solution.